lib/config/index.js

```diff
@@ -83,6 +89,12 @@ async function parseConfigs(env, argv) {
         config.token,
         config.endpoint
       );
+    } else if (config.platform === 'vsts') {
+      logger.info('Autodiscovering vsts repositories');
+      config.repositories = await vstsApi.getRepos(
```

Comment:
This is not currently implemented. Is it possible? Or should we make a future issue for it even if so?

Reply:
Done